hex/Hex2.cs

```diff
@@ -163,12 +173,16 @@ class Hex2 : UserControl {
             var charWidth = (int)(Font.Size * 2);
             var asciiStart = charWidth * numColumns;
             rect = new Rectangle(col * charWidth, row * 24, charWidth, 24);
-            Invalidate(rect);
+            Invalidate(rect);       
```

Comment:
Oops sorry about that, didn't know that extra whitespace would be counted as a change.